cost/azure/idle_compute_instances/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v4.6
+
+- Added hash_exclude to ignore changes to the averageCPU, averages and savings, since those will change constantly
```

Comment:
This still refers to much to the actual code. Keep in mind that the CHANGELOG needs to be understandable to someone who is using this policy but never looks at the actual policy template code. For example, we shouldn't mention hash_exclude at all, or refer to entries in the export statement by their code labels, since an end user would never see this stuff.

I also don't see a field called "averages" in the export statement, so I am not sure why that is mentioned or included in the hash_exclude.

An example of a better CHANGELOG entry for this:

 - New escalations are no longer created if Estimated Monthly Savings or CPU Average have changed, but nothing else has.

Reply:
Thank you, changed